WTWSMS/history/characters/WTWSMS_enlil.txt

```diff
@@ -1425,7 +1423,7 @@
 	religion = "germanic_pagan"
 	culture = "old_frankish"
 	father = 36096
-	400.1.1 = {
+	380.1.1 = {
```

Comment:
Should 36096 be his father in the first place? Geni says:

> Chronicle of Fredegar (7th century) says his father was Theudemeres, one of the leaders of the Salian Franks and king of Thérouanne (409–414).
Liber Historiae Francorum (8th century) says his father was Pharamond, perhaps a legendary figure.

Pharamond is 36099 and Theudemeres is 36105. We should probably change Chlodio's father to one of them. What do you think @loup99?

Reply:
Well, 36096  is Mallobaudes: https://en.wikipedia.org/wiki/Mallobaudes In the mod he is father to 36097, 36098, 36099 and 36100. 36100 being born in 380 does make a lot more sense if you don't want to redo the whole setup.

Reply:
But him being son of Mallobaudes doesn't seem to be mentioned anywhere, and making him son of Pharamond (who **is** mentioned as his father in one of the sources) wouldn't require changing his birth date. Double gain 😀

Reply:
Then you have to change the whole family, because otherwise you break a lot of ties. Probably possible for the future, but not really something that should block this PR.

Reply:
OK, so if it's a temporary fix we should keep the original date in a comment.